llvm/lib/Analysis/DependenceAnalysis.cpp

```diff
@@ -3649,6 +3646,111 @@ DependenceInfo::depends(Instruction *Src, Instruction *Dst) {
     LLVM_DEBUG(dbgs() << "can't analyze SCEV with different pointer base\n");
     return std::make_unique<Dependence>(Src, Dst);
   }
+
+  auto EltSize = SrcLoc.Size.toRaw();
```

Comment:
[style] [No Almost-Always-Auto](https://llvm.org/docs/CodingStandards.html#use-auto-type-deduction-to-make-code-more-readable)